src/odemis/acq/stream/_helper.py

```diff
@@ -1448,9 +1450,161 @@ def __init__(self, name, detector, dataflow, emitter, **kwargs):
         emtvas: don't put resolution or scale
         """
         if "acq_type" not in kwargs:
-            kwargs["acq_type"] = model.MD_AT_EM
+            kwargs["acq_type"] = model.MD_AT_EBIC
```

Comment:
It is a bit unclear to me why this is the fallback now, does it matter? Do we rely on it?

Reply:
We didn't have an office "EBIC" acquisition type until now. So we had been lazy and claim that EBIC is the same as SEM. This PR add the EBIC acquisition type, and here, we retro-actively use it on the (old) standard EBIC stream. Note that it's not very used in Odemis. Mostly for the icon in the merge slider.